### DISCUSSION TITLE
Adjacent time interval constraint of operations in a demand

### DISCUSSION THREAD

**Discussion by binghui1992** [frePPLe/frepple#585] [Q&A] [answered]
I'm quite new to FrePPle, and now I'm trying to use it to schedule a series of tasks. In fact it is a series of orders, each order contains one or multiple tasks, in case of one order with multiple tasks, the tasks have a precedent cosntaint, e.g. task A must be completed before task B can start. In addition to the precedent constraint between the tasks in an order, some cases require the tasks should be shceduled in a adjacent time interval, e.g. task A finished, and task B starts immediately, meaning there is no time interval between those two, and other cases do not have such requirement.

I knew FrePPle is for a manufactering industry use, however I wonder if FrePPle provides functionality with tasks scheduling which does not consume material nor produce product (indeed I could achieve this by introducing a virtual material) and with such adjacent time interval constraint of operations?

**Comment by jdetaeye** [comment, marked as answer]
See https://github.com/frePPLe/frepple/discussions/524#discussioncomment-5846932
